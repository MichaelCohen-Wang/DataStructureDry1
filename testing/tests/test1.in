add_ship 6299 13
remove_ship 6299
remove_ship 138509
add_ship 202291 6
remove_ship 202291
add_ship 177812 29
add_ship 175213 25
remove_ship 175213