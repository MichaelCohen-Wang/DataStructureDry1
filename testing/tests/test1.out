add_ship: SUCCESS
remove_ship: SUCCESS
remove_ship: FAILURE
add_ship: SUCCESS
remove_ship: SUCCESS
add_ship: SUCCESS
add_ship: SUCCESS
remove_ship: SUCCESS
